tests/framework/PostProcessors/InterfacedPostProcessor/HStoPSoperators/lorentzAttractor.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+''' from wikipedia: dx/dt = sigma*(y-x)  ; dy/dt = x*(rho-z)-y  dz/dt = x*y-beta*z  ; '''
+
+import numpy as np
+#import pylab as pyl
+#import random
+#import mpl_toolkits.mplot3d.axes3d as p3
+
+def initialize(self,runInfoDict,inputFiles):
+  self.sigma = 10.0
+  self.rho   = 28.0
+  self.beta  = 8.0/3.0
+  return
+
+def run(self,Input):
+  max_time = 1.0
+  t_step = 0.1
+
+  numberTimeSteps = int(max_time/t_step)
+
+  self.x = np.zeros(numberTimeSteps)
+  self.y = np.zeros(numberTimeSteps)
+  self.z = np.zeros(numberTimeSteps)
+  self.time = np.zeros(numberTimeSteps)
+
+  self.x0 = Input['x0']
+  self.y0 = Input['y0']
+  self.z0 = Input['z0']
+
+  self.x[0] = Input['x0']
+  self.y[0] = Input['y0']
+  self.z[0] = Input['z0']
+  self.time[0]= 0
+
+  for t in range (numberTimeSteps-1):
+    self.time[t+1] = self.time[t] + t_step
+    self.x[t+1]    = self.x[t] + self.sigma*(self.y[t]-self.x[t]) * t_step
+    self.y[t+1]    = self.y[t] + (self.x[t]*(self.rho-self.z[t])-self.y[t]) * t_step
+    self.z[t+1]    = self.z[t] + (self.x[t]*self.y[t]-self.beta*self.z[t]) * t_step
+
+    #self.ax.plot3D(self.x,self.y,self.z)
+
+    #pyl.savefig('test.png')
```

Comment:
We may have a unique place to keep our models. Currently we have many copies of this model.

Reply:
agree